src/coreclr/inc/corinfo.h

```diff
@@ -2983,6 +2984,8 @@ class ICorStaticInfo
     // Obtains a list of exact classes for a given base type. Returns 0 if the number of
     // the exact classes is greater than maxExactClasses or if more types might be loaded
     // in future.
+    // When called with maxExactClasses == 0, returns CORINFO_NEVER_INSTANTIATED if an
+    // exact class of this type cannot exist. Returns 0 otherwise.
     virtual int getExactClasses(
```

Comment:
Perhaps -1 should mean "I don't know" and 0 means that there are 0 exact classes that can be instanciated?
An alternative is to change the API to be `bool getExactClasses(..., int* count)`

Reply:
> This should ideally just be a separate JitInterface method and I would have done it if it was a two-liner change.

Regardless this is changing the JIT-EE interface so it should bump the JIT-EE GUID.
I agree it seems better to introduce a new JIT-EE method to avoid the complication of the contract.

Reply:
I made this "-1": don't know, or too many, "0": exactly zero. It didn't complicate the existing uses as much as I thought.